doc/manuals/version.rst

```diff
@@ -0,0 +1,5 @@
+.. |version| unicode:: 1.1.0
+   :trim:
+
+.. |space| unicode:: 0xA0
```

Comment:
Why is it necessary to trim whitespace and then add it back in when needed?

If you don't use `:trim:` and write `v|version|` do you get `v1.1.0`, `v 1.1.0`, or does it just fail to replace without whitespace before the `|`?

Reply:
When inserting replacement text or unicode, you must put spaces on either side. v|version| results in "v|version|" because it's not recognized as an insertion. As a result, any inserted text has spaces on either side, which isn't always what we want, so I trimmed the whitespace and added it back when necessary.

Reply:
Got it.  Thanks